charts/gardener-extension-shoot-rsyslog-relp-admission/values.yaml

```diff
@@ -9,7 +9,7 @@ global:
     user:
       name: ""
   image:
-    repository: eu.gcr.io/gardener-project/gardener/extensions/shoot-rsyslog-relp-admission
+    repository: europe-docker.pkg.dev/gardener-project/public/gardener/extensions/shoot-rsyslog-relp-admission
```

Comment:
Builds on head updates are preformed by this job https://github.com/gardener/ci-infra/blob/master/config/jobs/gardener-extension-shoot-rsyslog-relp/gardener-extension-shoot-rsyslog-relp-build-images.yaml instead of concourse. That one should be adapted since the images from head updates are used during release validation.

Reply:
@plkokanov : https://github.com/gardener/ci-infra/pull/1067